fr.yml

```diff
@@ -62,367 +62,365 @@ tray:
-      expandLines_many: "Développer {{count}} lignes cachées"
+      title:  "Modifications du code source"
+      noData: "La version installée est identique à la version sélectionnée."
+      splitView: "Vue divisée"
+      expandLines_one: "Développer une ligne masquée"
+      expandLines_other:  "Développer {{count}} lignes masquées"
   modSearch:
     placeholder: "Rechercher des mods..."
-    noResults: ~
+    noResults: "Aucun mod ne correspond aux filtres actuels"
   home:
     browse: "Parcourir les mods"
-    filter:
-      enabled: ~
-      disabled: ~
-      updateAvailable: ~
-      clearFilters: ~
-    installedMods:
+    filter: 
+      enabled: "Mods activés"
+      disabled: "Mods désactivés"
+      updateAvailable: "Mods avec mise à jour disponible"
+      clearFilters: "Effacer les filtres"
+    installedMods: 
       title: "Mods installés"
       noMods: "Aucun mod n'est installé"
       grid:
-        name: ~
-        description: ~
-        author: ~
-        version: ~
-        status: ~
-    featuredMods:
-      title: "Mods mis en avant"
-      noMods: "Il n'y a aucun mod mis en avant qui n'ait pas été installé"
-      explore: "Explorer les autres mods"
+        name: "Nom"
+        description: "Description"
+        author: "Auteur"
+        version:  "Version"
+        status: "Statut"
+    featuredMods: 
+      title: "Mods en vedette"
+      noMods: "Il n'y a pas de mods en vedette qui n'ont pas encore été installés"
+      explore: "Explorer d'autres mods"
   explore:
-    search:
-      popularAndTopRated: "Populaires et mieux notés"
+    search: 
+      popularAndTopRated: "Populaires et les mieux notés"
       popular: "Populaires"
-      topRated: "Mieux notés"
-      newest: "Récents"
-      lastUpdated: "Mis a jour récemment"
+      topRated: "Les mieux notés"
+      newest: "Les plus récents"
+      lastUpdated: "Dernière mise à jour"
       alphabeticalOrder: "Ordre alphabétique"
     filter:
-      installationStatus: ~
-      installed: ~
-      notInstalled: ~
-      author: ~
-      process: ~
-      showMore: ~
-      clearFilters: ~
+      installationStatus: "Statut d'installation"
+      installed: "Installé"
+      notInstalled: "Non installé"
+      author: "Auteur du mod"
+      process: "Processus ciblé"
+      showMore: "Afficher plus..."
+      clearFilters: "Effacer les filtres"
   settings:
-    language:
+    language: 
       title: "Langue"
-      description: "Sélectionnez votre langue préférée pour Windhawk."
-      contribute: "<0>Contribuer pour une nouvelle traduction</0>."
-      credits: "Traduction française par <0>Jeremy Clement</0>."
-      creditsLink: "mailto:clement.jeremy.pro@outlook.fr"
+      description: "Sélectionnez votre langue d'affichage préférée pour Windhawk."
+      contribute: "<0>Contribuez une nouvelle traduction</0>."
+      credits: "Traduction française par <0>John Smith</0>."
+      creditsLink: "mailto:john. smith@example.com"
```

Comment:
Please adjust the translation credits. Currently it specifies the placeholder name, John Smith.